src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -93,6 +105,9 @@ public SwerveSubsystem() {
         zeroHeading();
         controlOrientationIsFOD = true;
 
+        //Add coast mode command to shuffleboard
+        configTab.add(new StartEndCommand(this::coastModules, this::brakeModules, this));
```

Comment:
Does this display well in shuffleboard? There's no assigned labels so I wonder how this looks in the GUI

Reply:
Forgot to push some commits, it displays fine in the newest version